### PR TITLE
Added entry point for different hardwares

### PR DESCRIPTION
**Pull request by PrabhanshuAttri**

### CODE REVIEW

File: hardware/CommunicationsPi/radio_transceiver.py [hunk 1]
```diff
@@ -31,18 +31,24 @@ def __init__(self, log_file_name=None, port=None):
...
+            self.port_pid = port_info.pid if hasattr(port_info, "pid") else None
+            self.port_vendor = (
+                port_info.manufacturer if hasattr(port_info, "manufacturer") else None
+            )
+            self.port_intf = (
+                port_info.interface if hasattr(port_info, "interface") else None
+            )
+            self.port_serial_number = (
+                port_info.serial_number if hasattr(port_info, "serial_number") else None
+            )
```
Comment (ab7289) [inline hardware/CommunicationsPi/radio_transceiver.py:44]: 👍

File: hardware/tests/test_web_client.py [hunk 2]
```diff
@@ -4,9 +4,8 @@
 from requests.exceptions import HTTPError
```
Comment (ab7289) [inline hardware/tests/test_web_client.py:5]: this is minor, but please update the class LanClientTests to be renamed WebClientTests so that it matches.

Comment (PrabhanshuAttri) [reply hardware/tests/test_web_client.py:5]: My bad. Fixed

Comment (PrabhanshuAttri) [reply hardware/tests/test_web_client.py:5]: Fixed.

**Pull request merged**